go/oasis-test-runner/scenario/e2e/gas_fees_staking.go

```diff
@@ -190,6 +190,22 @@ func (sc *gasFeesImpl) runTests(ctx context.Context) error {
 		return err
 	}
 
+	// As of the last time this comment was updated:
```

Comment:
Maybe should add a sanity check that this is indeed the case by adding a manually computed with this comment. 

Would also help with keeping this updated. 

Also note that this test is also run with a dump-restore variant so in the second run the expected result will differ.

Reply:
what happens in the dump-restore variant?

Reply:
i think the `last_block_fees` & `common_pool` in the dumped genesis will be empty and the rest stays the same?

Reply:
https://github.com/oasislabs/oasis-core/pull/2798

Reply:
thanks!